docs/learn/learn-nft-pallets.md

```diff
@@ -0,0 +1,103 @@
+---
+id: learn-nft-pallets
+title: NFT Pallets
+sidebar_label: NFT Pallets
+description: An overview of the functionalities of NFT Pallets in Polkadot Ecosystem.
+keywords: [NFT, non-fungible token, NFT 2.0, nfts, NFT pallets]
+slug: ../learn-nft-pallets
+---
+
+NFT [pallets](http://localhost:3000/docs/learn-extrinsics#pallets-and-extrinsics) allow substrate
+developers to easily implement NFT-related actions within their dApp.
+
+## NFTS Pallet
+
+### Current Implementations
+
+Nfts is a [FRAME pallet](https://polkadot.js.org/docs/substrate/extrinsics#nfts) currently deployed
+on Westmint that provides a multitude of functions to interact with NFTs.
+
+The pallet comes with a new way to configure it, as well as configure collections and items.
+Pallet-level [feature flags](https://github.com/paritytech/substrate/pull/12367) allow to disable
+functionalities that are not needed in the runtime.
+
+There are features that common to the nft world such as:
+
+- Buying an item up for sale.
+- Creating (i.e. mint) and burn (i.e. destroy) a collection of items or a single item (burning must
+  be signed either by the admin of the collection or the owner). The creation of an item usually
+  involves setting some attributes specific to that item (an this is actually what drives the value
+  of the item). The creation of a collection also involves the specification of the maximum number
+  of items the collection has (an this also will drive the value of the collection's items in the
+  future).
```

Comment:
Not all NFTs are meant for value creation. For instance, Soul bound NFTs, POAPs. 

```suggestion
- Buying an item up for sale.
- Creating (i.e., mint) and burning (i.e., destroy) a collection of items or a single item (burning must
  be signed either by the admin of the collection or the owner). Creating an item usually
    involves setting some attributes specific to that item. Creating a collection also requires the specification of the maximum number of items.
```